Rubberduck.Refactorings/MoveFolder/IMoveMultipleFoldersPresenter.cs

```diff
@@ -0,0 +1,5 @@
+﻿namespace Rubberduck.Refactorings.MoveFolder
+{
+    public interface IMoveMultipleFoldersPresenter : IRefactoringPresenter<MoveMultipleFoldersModel>
+    {}
```

Comment:
Why do we need this marker interface?

Reply:
Because we use it as the type parameter in the presenter and dialog setup.